tests/rust/body.toml

```diff
@@ -18,3 +18,25 @@
 "MyUnion" = """
   int32_t extra_member; // yolo
 """
+
+[export.body_prepend]
+"MyFancyStruct_Prepended" = """
+#ifdef __cplusplus
+  inline void prepended_wohoo();
+#endif
+"""
+
+"MyFancyEnum_Prepended" = """
+  #ifdef __cplusplus
+    inline void wohoo();
+  #endif
+"""
+
+"MyCLikeEnum_Prepended" = """
+  BogusVariantForSerializationForExample,
```

Comment:
This doesn't work because it wouldn't be legal C++, so yeah, it is intended. I guess you could add more variants, but that didn't seem useful. Could be fixed if you have some use case for it but...

Reply:
I brought it up because it was in the documentation + test of the existing body example. If it's intended to not work, should we remove the existing example? It's also mentioned in the documentation.